arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1142 @@
+            }
+          },
+          "202": {
+            "description": "Accepted response definition"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/search/{id}": {
+      "post": {
+        "tags": [
+          "Search"
+        ],
+        "operationId": "Search_UpdateSearchResults",
+        "description": "Gets updated search results for a given search query.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Resource Group name of the Log Analytics Workspace"
+          },
+          {
+            "name": "workspaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Log Analytics workspace name"
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the search that will have results updated."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition.",
+            "schema": {
+              "$ref": "#/definitions/SearchResultResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Account": {
+      "properties": {
+        "customerId": {
+          "type": "string",
+          "description": "Gets or sets the unique ID of the account."
+        },
+        "accountName": {
+          "type": "string",
+          "description": "Gets or sets the name of the account."
+        },
+        "workspaceName": {
+          "type": "string",
+          "description": "Gets or sets the DNS valid workspace name associated with the account."
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets the Azure region that the account was created in."
+        }
+      },
+      "description": "Metadata for an Operational Insights account that isn't linked to a workspace."
+    },
+    "Tag": {
+      "properties": {
+        "Name": {
+          "type": "string",
+          "description": "Gets or sets the tag name."
+        },
+        "Value": {
+          "type": "string",
+          "description": "Gets or sets the tag value."
+        }
+      },
+      "description": "A tag of a saved search."
+    },
+    "SavedSearchProperties": {
+      "properties": {
+        "Category": {
+          "type": "string",
+          "description": "Gets or sets the category."
```

Comment:
This is not a enum. It can be any string the user wants to use to group the saved searches. The UI allows the user to find a saved search faster by choose the category first. 
I changed the description to be more specific